core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockStore.java

```diff
@@ -86,8 +86,8 @@ public RocksBlockStore(String baseDir) {
     if (!FileUtils.exists(dbPath)) {
       try {
         FileUtils.createDir(dbPath);
-      } catch (IOException ioe) {
-        LOG.warn("Failed to create nonexistent db path at: {}. Error:{}", dbPath, ioe);
+      } catch (IOException e) {
+        LOG.warn("Failed to create nonexistent db path at: {}. Error:{}", dbPath, e);
```

Comment:
do we need `e.toString` here?

Reply:
It's implicit I think, because it's interpreted as `Object`

Reply:
I was worried about it treating it differently when the last arg is a throwable, but looking at the implementation I think you are right.